snippets/csharp/System.Linq/ILookupTKey,TElement/Overview/ILookup.cs

```diff
@@ -1,33 +1,30 @@
 ﻿using System;
-using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
 
 namespace SequenceExamples
 {
     static class ILookup
     {
-        static void Main(string[] args)
-        {
-            ILookupExample();
-        }
+        static void Main() => ILookupExample();
```

Comment:
This whole sample would look much cleaner with top-level statements and the global namespace.